app/routes/posts.js

```
@@ -0,0 +1,10 @@
+import Ember from 'ember';
+import AuthenticatedRouteMixin from 'ember-simple-auth/mixins/authenticated-route-mixin';
+
+export default Ember.Route.extend(AuthenticatedRouteMixin, {
+  model() {
+    return this.store.findAll('post').catch(() => {
+      this.transitionTo('index');
```

This just swallows all errors that might occur when loading the model. I wouldn't handle rejection at all here.